src/runtime_src/xocl/api/plugin/xdp/profile.cpp

```diff
@@ -215,6 +220,22 @@ void get_address_bank(cl_mem buffer, uint64_t &address, std::string &bank)
   }
 }
 
+// Attempt to find if two buffers are resident on the same device
```

Comment:
@stsoe - We need to differentiate between CDMA and P2P buffer transfers. Is this a good method to do that? I realize clEnqueueCopyBuffer does not yet support P2P transfers, but this is in anticipation of that.